MyXamarinApp.API/Profiles/ApplicationMapper.cs

```diff
@@ -1,14 +1,27 @@
 ﻿using AutoMapper;
 using MyXamarinApp.API.Data;
 using MyXamarinApp.API.Models;
+using System;
+using System.Globalization;
 
 namespace MyXamarinApp.API.Profiles
 {
     public class ApplicationMapper : Profile
     {
         public ApplicationMapper()
         {
-            CreateMap<EmployeeModel, Employee>().ReverseMap();
+            CreateMap<EmployeeModel, Employee>().ForMember(x => x.Salary, x => x.MapFrom(src => GetSalaryAsInt(src)));
```

Comment:
I am not sure I understand these. What are these doing ?

Reply:
converting the comma separated value(string) of salary to int.